### PR TITLE
Remove atoms

### LINKED ISSUE #66: Remove Atoms from python CSV file
This is a regression from the last release

## Expected Behavior
`atom` should be ignored

## Current Behavior
`atom` tokens are displayed as `Atoms`

### PR DESCRIPTION
**Pull request by dane-johnson**
## Description

Removes the atom rules

## Motivation and Context

Atoms should be ignored.
closes #66 

## Checklist:

- [x] All new and existing tests passed.

### CODE REVIEW

File: mappings/python3.csv [hunk 1]
```diff
@@ -64,7 +64,7 @@ term,0,,
 factor,0,,
 power,0,,
 trailed_atom,0,,
-atom,1,Atoms,#ABDBF7
+atom,0,Atoms,#ABDBF7
```
Comment (thebho) [inline mappings/python3.csv]: Should we removed the friendly name and color too?

**Review by thebho — CHANGES_REQUESTED** [review]
Also, this links to #66

File: mappings/python3.csv (hunk 1 above)
Comment (dane-johnson) [reply mappings/python3.csv]: Doesn't matter, those fields are ignored if the token is

Comment (thebho) [reply mappings/python3.csv]: I know it "doesn't matter", but when all the other ignored rules are one way, we should keep it consistent

Comment (dane-johnson) [reply mappings/python3.csv]: 🤷‍♂️

**Review by thebho — APPROVED** [review]
Gratzie

**Pull request merged**